src/query/sql/src/planner/semantic/type_check.rs

```diff
@@ -2513,8 +2513,24 @@ impl<'a> TypeChecker<'a> {
                     .await
             }
             ("greatest", args) => {
+                if args.is_empty() {
+                    return None;
+                }
```

Comment:
It is unnecessary to forbid zero-arg `greatest` since `array_max([])` returns `NULL`, therefore `greatest()` should also return `NULL`.

Reply:
@andylokandy can't find the function, could you give me some advice ?

Reply:
`args.is_empty()` is allowed, just remove      this check.